libass/ass.c

```diff
@@ -1480,6 +1464,19 @@ ASS_Track *ass_new_track(ASS_Library *library)
         free(track);
         return NULL;
     }
+    int def_sid = ass_alloc_style(track);
+    if (def_sid < 0) {
+        free(track->parser_priv);
+        return NULL;
```

Comment:
This is still missing a `free(track)` :-)

Reply:
Thanks.